### PR TITLE
Use versioned reference to action

### PR DESCRIPTION
**Pull request by kkadosa**
Removed steps so the workflow actually works
Updated readme slightly to clarify what exactly the action does

### CODE REVIEW

**Review by dandrid — CHANGES_REQUESTED** [review]

File: readme.md [hunk 1]
```diff
@@ -5,9 +5,9 @@
 This repository contains a GitHub Action for deploying and running the IncQuery AUTOSAR-UML Bridge.
  The repository also contains a workflow example demonstrating how to use the action.
 
-## What does this action do?
+## What does the workflow do?
```
[the remark this answers is not in the document]
Comment (kkadosa) [reply readme.md:8]: The quotation literally contains that.

[the remark this answers is not in the document]
Comment (dandrid) [reply readme.md:8]: We have an action, which is reusable, which:

1. Downloads and installs the IncQuery AUTOSAR-UML Bridge, if it is not already installed
2. Runs the IncQuery AUTOSAR-UML Bridge

We have an example workflow, which demonstrates a potential usage of the AUTOSAR-UML Bridge Action:

1. Checks out the Git repository
3. Backups up the EA Project File
4. Runs IncQuery AUTOSAR-UML Bridge Action
5. Pushes the updated model

The users would mainly like to reuse the the Action (which is about how to run the AUTOSAR-UML Bridge). 

The current section mixes both the Workflow and the Action. I think we should separate these. In the first section, we could add some details on how the AUTOSAR-UML Bridge Action works, encouraging the users to reuse the Action in their own Workflows. Then we could have an Example Workflow section, which details how the example works, and shows a usage example for the AUTOSAR-UML Bridge Action.

[the remark this answers is not in the document]
Comment (kkadosa) [reply readme.md:8]: No, the current section deals purely with the workflow. It is a subsection in the section dealing with the action. The basic description of the action is directly above this change, within the image, and is then followed by more technical sections.

File: readme.md [hunk 2]
```diff
@@ -5,9 +5,9 @@
...
 
-Firslty, this action downloads and installs **IncQuery AUTOSAR-UML Bridge**
+Firslty, using the provided action, the workflow downloads and installs **IncQuery AUTOSAR-UML Bridge**
  (unless already installed on the GitHub Runner), then executes a transformation on an input AUTSOAR model,
```
Comment (dandrid) [inline readme.md:11]: AUTSOAR

**Review by dandrid — APPROVED** [review]

**Pull request merged**